packages/edit-site/src/components/block-editor/style.scss

```diff
@@ -106,39 +106,35 @@
 
 	&::after {
 		position: absolute;
-		top: $grid-unit-30;
+		top: $grid-unit-20;
 		left: $grid-unit-05;
 		right: 0;
-		bottom: $grid-unit-30;
+		bottom: $grid-unit-20;
 		content: "";
 		width: $grid-unit-05;
-		background: $gray-600;
+		background-color: rgba($gray-700, 0.4);
 		border-radius: $radius-block-ui;
 	}
 
 	&.is-left {
-		left: -$grid-unit-20;
+		// Subtract half of the handle width to properly center.
+		left: -$grid-unit-20 - ($grid-unit-05 / 2);
```

Comment:
This notation will alert us to a warning error when starting or building the project.

![start-error](https://github.com/WordPress/gutenberg/assets/54422211/f81cb8aa-648d-4191-81eb-895f89f3adfe)

You'll probably need to change it to something like this:

```
left: -$grid-unit-20 - math.div($grid-unit-05, 2);
```

Reply:
Thanks for pointing that out @t-hamano - I created a PR for that [here](https://github.com/WordPress/gutenberg/pull/61285).